app/lib/pre_assembly/batch.rb

```diff
@@ -116,7 +116,7 @@ def containers_via_manifest(&block)
     # Takes a root path of the object folder (as supplied in the object manifest).
     # It then finds all files within with an eager glob pattern.
     def discover_items_via_crawl(root)
-      Dir.glob("#{root}/**/*")
+      Dir.glob("#{root}/**/*").select { |fname| File.file?(fname) }
     end
 
```

Comment:
Another critical change, we want to discover only files at all hierarhcies of the object (this would previously incorrectly select folders in the list as well, which we don't need)